packages/gestalt/src/Masonry.js

```diff
@@ -56,12 +55,10 @@ type Props<T> = {|
   loadItems?:
     | false
     | ((
-        // eslint-disable-next-line flowtype/require-exact-type
-        ?{
+        ?{|
           from: number,
-        }
-        // eslint-disable-next-line flowtype/require-exact-type
-      ) => void | boolean | {}),
+        |}
+      ) => void | boolean | { ... }),
```

Comment:
changing from exact to inexact since consumers may be returning an object. although this should have meant `{||}` because of the `exact_by_default` option, flow running in downstream projects may not correctly respect that, so it's safer to change to make this inexact.

in the future, we could just change the return type to `void` because the return value isn't used at all.